src/SFA.DAS.FAT.Web/Views/Shared/_ValidationSummary.cshtml

```diff
@@ -12,14 +12,16 @@
                     foreach (var property in explorer.Properties)
                     {
                         var propertyName = property.Metadata.PropertyName;
+                        var propertyNameLink = propertyName == "Location" ? "search-location" : propertyName;
```

Comment:
This just does not read right to me, why are we handling one specific property in this generic piece of code?

Reply:
All the dropdowns for location are called id 'search-location' to make the javascript work, whilst still using the field name 'Location'.  Validation summary linkages work through names, not ids, and expect id and name to be the same, from the look of it. I tried renaming but javascript did not play nice, and it means either renaming it for all location dropdowns, or putting in two lots of javascript - and I'm not sure that will work right.  I tried the former, and it just all stopped working, so I eventually reverted after about 30 minutes of trying to get it working. As the issue is merely make the validation summary anchor point to the right anchor, the above addresses that issue.

I will leave as is, but anything else is probably a lot more work for little benefit.